lib/etl/extract/process_files_in_s3_bucket_folders.rb

```diff
@@ -0,0 +1,52 @@
+require 'aws-sdk'
+require 'tempfile'
+
+class ProcessFilesInS3BucketFolders
+  def initialize(bucket_name, prefix_folders, file_ext)
+    @bucket_name = bucket_name
+    @prefix_folders = prefix_folders
```

Comment:
Taking this a step further, if we change this just to accept one file, perhaps it could actually just be a modification to [`ProcessAllFilesInS3Bucket`](https://github.com/greencommons/commons/blob/master/lib/etl/extract/process_all_files_in_s3_bucket.rb)

What do you think?